Libraries/LibWeb/CSS/StyleComputer.cpp

```diff
@@ -1041,7 +1041,7 @@ void StyleComputer::collect_animation_into(DOM::Element& element, Optional<CSS::
                 [&](RefPtr<CSSStyleValue const> value) -> RefPtr<CSSStyleValue const> {
                     if (value->is_unresolved())
                         return Parser::Parser::resolve_unresolved_style_value(Parser::ParsingContext { element.document() }, element, pseudo_element, it.key, value->as_unresolved());
-                    return value;
+                    return compute_property_value(element, it.key, value.release_nonnull());
```

Comment:
Computing the property values here avoids the crash but its likely doing more work than is needed. Ideally, the property values are calculated when the target element is set on the effect somewhere [here](https://github.com/LadybirdBrowser/ladybird/blob/master/Libraries/LibWeb/CSS/StyleComputer.cpp#L1563).